backend/blueprints/spa_api/spa_api.py

```diff
@@ -751,3 +752,60 @@ def api_admin_get_logs(query_params=None):
 @with_query_params(accepted_query_params=[QueryParam(name='id', type_=str, optional=False)])
 def api_admin_get_replay(query_params=None):
     return redirect(f"https://storage.googleapis.com/{FAILED_BUCKET}/{query_params['id']}.replay")
+
+
+# GROUPS
+
+@bp.route('/groups/add', methods=['POST'])
```

Comment:
can we maybe make groups requests its own file instead of putting it in spa_api?

As I do eventually want to make every blueprint product logically in its own file.  It would be nice to start this trend so we have less to eventually break up

Reply:
Nested blueprints [are not supported in Flask](https://stackoverflow.com/a/33005066/2352847) so this requires a good amount of engineering effort that is outside the scope this PR.